source/index.rst

```diff
@@ -11,7 +11,184 @@
 FIRST Robotics Competition Control System
 ===========================================
 
-Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! An overview of the changes from 2020 to 2021 is available on the :ref:`New for 2021 <docs/yearly-overview/yearly-changelog:New for 2021>` document.
+Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site will contain everything you need to know for building and programming a competition robot!
+
+Community translations can be found for a variety of languages in the bottom-left menu.
+
+.. panels::
+   :column: col-sm-6 d-flex pb-3
+   :footer: bg-light
+
+   ---
+   :header: bg-success font-weight-bold text-white
+
+   Returning Teams
+
+   ^^^
+
+   If you are a returning team, please check out the overview of changes from 2020 to 2021.
+
+   +++
+
+   .. link-button:: /docs/yearly-overview/yearly-changelog
+      :type: ref
+      :text: Go to Changelog
+      :classes: btn-outline-primary btn-block
+
+   ---
+   :header: bg-info font-weight-bold text-white
+
+   New Teams
+
+   ^^^
+
+   Please read through the Zero-to-Robot tutorial! This will guide you through preparation, building and programming your robot!
```

Comment:
```suggestion
   Please read through the Zero-to-Robot tutorial! This will guide you through preparation, building and programming a basic robot!
```